src/constants/Blacklist.js

```diff
@@ -1,40 +1,41 @@
 /**
  * An array of regex swears.
+ * @type {[string, number][]}
  */
 const swearWords = [
-    'arse',
-    'anus',
-    'boob',
-    'bitch',
-    'bastard',
-    'blowjob',
-    'boner',
-    'cock',
-    'clit',
-    'cunt',
-    'cum',
-    'dick',
-    'dildo',
-    'ejaculate',
-    'motherfucker',
-    'fuck',
-    'fucker',
-    'fag',
-    'gangbang',
-    'horny',
-    'jackoff',
-    'jerkoff',
-    'masturbate',
-    'nigga',
-    'nigger',
-    'porn',
-    'pussy',
-    'shit',
-    'sex',
-    'slut',
-    'tits',
-    'titties',
-    'vagina',
+    ['arse', 1],
```

Comment:
use a map instead. then you can later just use .keys() for filtering which is easier to understand when looking at it than the double .filter() thing and extracting only the word from the array